alert-common/build.gradle

```diff
@@ -2,7 +2,7 @@ ext.moduleName = 'com.synopsys.integration.alert.common'
 
 dependencies {
     api 'com.synopsys.integration:issuetracker-jira:0.0.3'
-    api 'com.blackducksoftware.integration:blackduck-common:45.0.6'
+    api 'com.synopsys.integration:blackduck-common:45.1.1-SNAPSHOT'
```

Comment:
Do we want a snapshot for now?

Reply:
I'm thinking no.

Reply:
oops that's my bad I missed that one.  Will update